### ISSUE TITLE
.kap files, OSM no longer georeferenced

### ISSUE THREAD

**Issue by SV-Pelagia** [qgis/QGIS#39630]
Never had a problem with this until last week or so:

When I bring in a .kap file as a raster layer, it comes in without any georeference (CRS?). I can zoom to layer to see it, but its lat/lon coordinates are nonsensical.

Same .kap files work fine in chartplotting software such as OpenCPN.
(I even downloaded  fresh new test kap/bsb files from NOAA -- still no luck 

File comes in with "Unknown CRS"

Name | 12323_1
-- | --
Path | C:\CHARTS\RASTER\TEMP\BSB_ROOT\12323\12323_1.KAP
CRS | Unknown CRS: PROJCRS["Global Mercator",BASEGEOGCRS["WGS 84",DATUM["World Geodetic System 1984",ELLIPSOID["WGS 84",6378137,298.257223563,LENGTHUNIT["metre",1]]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433]],ID["EPSG",4326]],CONVERSION["unnamed",METHOD["Mercator (variant B)",ID["EPSG",9805]],PARAMETER["Latitude of 1st standard parallel",0,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8823]],PARAMETER["Latitude of natural origin",0,ANGLEUNIT["degree",0.0174532925199433,ID["EPSG",9122]]],PARAMETER["Longitude of natural origin",-74,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8802]],PARAMETER["False easting",0,LENGTHUNIT["Meter",1],ID["EPSG",8806]],PARAMETER["False northing",0,LENGTHUNIT["Meter",1],ID["EPSG",8807]]],CS[Cartesian,2],AXIS["easting",east,ORDER[1],LENGTHUNIT["Meter",1]],AXIS["northing",north,ORDER[2],LENGTHUNIT["Meter",1]]] - Projected
Extent | -48810.0743519349925919,4755289.5712047731503844 : 39636.3257532024435932,4863264.4713630015030503
Unit | meters
Width | 13388
Height | 16344
Data type | Byte - Eight bit unsigned integer
GDAL Driver Description | BSB
GDAL Driver Metadata | Maptech BSB Nautical Charts
Dataset Description |  
Compression |  
Band 1 |  
More information |  
Dimensions | X: 13388 Y: 16344 Bands: 1
Origin | -48810.1,4.86326e+06
Pixel Size | 6.60639379333264376,-6.60639379333264376

**Comment by gioman** [comment]
@SV-Pelagia please attach/link a sample.

**Comment by SV-Pelagia** [comment]
http://www.charts.noaa.gov/RNCs/12323.zip

**Comment by gioman** [comment]
@SV-Pelagia the fact that the layer comes up with an unknown CRS it does not means it has no valid CRS,in fact here the layer works ok when adding a basemap 

![Screenshot_20201027_110249](https://user-images.githubusercontent.com/1951107/97293095-18550580-1844-11eb-9736-ba7a19fec06c.png)

it seems the CRS used by these maps is a custom one, so QGIS says is "unknown".

**Comment by SV-Pelagia** [comment]
Thanks gioman!
Your comment made me check again...

I tried setting OSM as the basemap. Wouldn't display correctly.
But then I noted the coordinates at the bottom were all wrong -- turns out it was set to incorrect EPSG.
Changed to EPSG 4326 and presto, the coordinates came out correct, and various".kap" files came in fine.

I THOUGHT I had previously set  the EPSG correct -- never had a problem with this. Perhaps some "update" to software? 

Whatever, it works now.
Thank you for responding quickly.
